units/provisioning_create_host.go

```diff
@@ -353,11 +353,25 @@ func (j *createHostJob) isImageBuilt(ctx context.Context) (bool, error) {
 		return false, errors.Errorf("parent for host '%s' not running", j.host.Id)
 	}
 	if ok := parent.ContainerImages[j.host.DockerOptions.Image]; ok {
+		grip.Info(message.Fields{
+			"message":  "image already exists, will start container",
+			"host":     j.host.Id,
+			"image":    j.host.DockerOptions.Image,
+			"attempts": j.CurrentAttempt,
+			"job":      j.ID(),
+		})
 		return true, nil
 	}
 
 	//  If the image is not already present on the parent, run job to build the new image
 	if j.BuildImageStarted == false {
+		grip.Info(message.Fields{
```

Comment:
should we further annotate the other logging statement in this if statement?

Reply:
It's not really a useful log message, but it is the only way that error is being handled so it's probably fine to keep as is

Reply:
All errors that get j.AddError() are logged when the job finished, which I think should reduce the need to log errors in jobs.

Reply:
That specific error is not added to the job because we don't want to retry if it happens, so we just log it